src/node_report.cc

```diff
@@ -48,6 +47,10 @@
 extern char** environ;
 #endif
 
+#ifndef MAXHOSTNAMELEN
+# define MAXHOSTNAMELEN 256
+#endif  // MAXHOSTNAMELEN
```

Comment:
Include `<netdb.h>` and `<sys/param.h>` to get the definition of `MAXHOSTNAMELEN`, see src/node_os.cc.